benchmark_runner/benchmark_operator/workload_flavors/perf_ci/hammerdb/internal_data/hammerdb_vm_template.yaml

```diff
@@ -74,14 +74,14 @@ spec:
       kind: vm
       client_vm:
         dedicatedcpuplacement: false
-        sockets: 1
-        cores: {{ cores }}
+        sockets: {{ vm_sockets }}
+        cores: {{ vm_cores }}
         threads: 1
         image: quay.io/cloud-bulldozer/{{ db_image }}
         limits:
-          memory: {{ limits_memory_GB }}Gi
+          memory: {{ vm_limits_memory_GB }}Gi
         requests:
-          memory: 10Mi
+          memory: {{ vm_requests_memory_GB }}Gi
         network:
           front_end: bridge # or masquerade
```

Comment:
Could you change to masquerade? It is the default and bridge is unsupported.

Reply:
Yes I will